src/pyj/read_book/view.pyj

```diff
@@ -475,10 +494,14 @@ class View:
             self.overlay.open_book()
         elif data.name is 'next':
             self.iframe_wrapper.send_message(
-                'next_screen', backwards=False, all_pages_on_screen=get_session_data().get('paged_margin_clicks_scroll_by_screen'))
+                'next_screen', backwards=False,
+                flip_if_rtl_page_progression=False,
+                all_pages_on_screen=get_session_data().get('paged_margin_clicks_scroll_by_screen'))
         elif data.name is 'previous':
```

Comment:
I actually cannot find where 'next' is sent. The only place that looks suspicious is toolbars.py, which seems like the reader UI that shows up on right-click, but I don't see a next page butter there.